src/Web/couch-potatoes-web-app/src/components/person/PersonSideBar.tsx

```diff
@@ -34,10 +35,29 @@ const PersonSideBar: FC<PersonProperties> =
          placeOfBirth,
          aliases
      }) => {
+        const [isOpen, setIsOpen] = useState(false);
+        const [expandedImage, setExpandedImage] = useState('');
+
+        const handleImageClick = (imageUrl: string) => {
+            setExpandedImage(imageUrl);
+            setIsOpen(true);
+        };
+
+        const handleModalClose = () => {
+            setExpandedImage('');
+            setIsOpen(false);
+        };
+
+
         return (
             <Card backgroundColor="transparent">
                 <CardBody>
-                    <Image src={uri} alt={alt} borderRadius='lg'/>
+                    <Image src={uri} alt={alt} borderRadius='lg'
+                           onClick={() =>
+                               handleImageClick(uri as string
```

Comment:
if you have a variable `uri` of type `string?` that you are sure never will be null, then you can do `uri!` instead of casting with `as`

Reply:
fixed